.storybook/__mocks__/react-native-gesture-handler.tsx

```diff
@@ -31,5 +32,6 @@ module.exports = {
      * to call Exclusive(doubleTap, singleTap).
      */
     Exclusive: () => undefined,
-  }
+  },
+  Switch: Switch
```

Comment:
Mocked using `react-native` due to issues with `react-native-gesture-handlers` component working in the Web